src/rhsm/connection.py

```diff
@@ -635,24 +635,35 @@ def _print_debug_info_about_request(self, request_type, handler, final_headers,
             else:
                 msg = blue_col + "Making request:" + end_col
             msg += red_col + " %s:%s %s %s" % (self.host, self.ssl_port, request_type, handler) + end_col
+            if self.proxy_hostname and self.proxy_port:
+                msg += blue_col + " using proxy " + red_col + f"{self.proxy_hostname}:{self.proxy_port}" + end_col
             if 'SUBMAN_DEBUG_PRINT_REQUEST_HEADER' in os.environ:
                 msg += blue_col + " %s" % final_headers + end_col
             if 'SUBMAN_DEBUG_PRINT_REQUEST_BODY' in os.environ and body is not None:
                 msg += yellow_col + " %s" % body + end_col
             print()
             print(msg)
             print()
-            if os.path.isdir('/tmp/sub-man') is False:
-                os.mkdir('/tmp/sub-man')
             if 'SUBMAN_DEBUG_SAVE_TRACEBACKS' in os.environ:
-                with tempfile.NamedTemporaryFile(
-                        dir='/tmp/sub-man',
-                        mode='w',
-                        prefix='traceback-',
-                        delete=False) as tmp_file:
-                    traceback.print_stack(file=tmp_file.file)
-                    print(green_col + '    traceback saved in: %s' % tmp_file.name + end_col)
-                    print()
+                debug_dir = Path("/tmp/rhsm/")
```

Comment:
Not a blocking comment but I don't think the final "/" is necessary here as we are referring to it as a directory in the very next statement.

Reply:
It's true that it's not. The trailing space is meant more as a visual clue than anything else. When you see a path with a trailing slash, you know it's a directory and you don't even have to read the variable name.

pathlib.Path is object oriented and does not care if the path ends with a slash or not.